web/js/layers/modal.js

```diff
@@ -467,6 +467,7 @@ export function layersModal(models, ui, config) {
 
         // Begin source level
         Object.values(current.sources).forEach(function (source) {
+          var showFullDescription;
```

Comment:
I might name this variable `isExpanded`, but that's subjective.

Reply:
Yea I tried to name this logically but I think `isExpanded` is more concise.